src/main/java/com/google/cloud/tools/appengine/experimental/deploy/DeployExample.java

```diff
@@ -19,13 +19,16 @@
 import com.google.cloud.tools.appengine.api.deploy.DefaultDeployConfiguration;
 import com.google.cloud.tools.appengine.cloudsdk.PathResolver;
 import com.google.cloud.tools.appengine.experimental.AppEngineRequestFactory;
+import com.google.cloud.tools.appengine.experimental.AppEngineRequestFuture;
 import com.google.cloud.tools.appengine.experimental.AppEngineRequests;
-import com.google.cloud.tools.appengine.experimental.internal.process.io.PrintStreamOutputHandler;
 
 import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
 import java.util.Collections;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.Future;
 
 public class DeployExample {
```

Comment:
Yeah this has kind of been in there to show us how to use it. It has no necessity for being there. It's not going to be publicly documented until we decide exactly what it is going to be.